components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -78,9 +78,17 @@
     private String passiveSTSWReply;
     private String openid;
     private String[] claimUris;
+    private List<InboundAuthenticationRequestConfig> customInbountAuthenticatorConfigs;
+    private List<String> standardInboundAuthTypes;
 
     public ApplicationBean() {
 
+        standardInboundAuthTypes = new ArrayList<String>();
+        standardInboundAuthTypes.add("oauth2");
+        standardInboundAuthTypes.add("wstrust");
+        standardInboundAuthTypes.add("samlsso");
+        standardInboundAuthTypes.add("openid");
+        standardInboundAuthTypes.add("passivests");
```

Comment:
We need to have this since when edit existing SP it send all inbound authenticators (ex. openid, passivests), then have to filter correct custom inbound authenticators